### PR TITLE
Flavio Changes and added contribs to data

### PR DESCRIPTION
**Pull request by SaraVieira**
@kutyel

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "now-build": "NODE_ENV=production npm run build:client -- -p && NODE_ENV=production npm run build:server -- -p",
     "build:client": "webpack --progress --config webpack/webpack.client.babel.js",
     "build:server": "webpack --progress --config webpack/webpack.server.babel.js",
+    "pretty": "prettier-standard {src,webpack}/**/*.js --single-quote",
```
Comment (SaraVieira) [inline package.json:13]: love this !

**Comment by SaraVieira** [comment]
https://graphql-portfolio-szfqfsrgvv.now.sh/

**Comment by kutyel** [comment]
Apparently everything works 👏🏼 Nice branch name btw 🤣

**Comment by SaraVieira** [comment]
It shall be merged and the website shall look way cleaner ! Thank you again 🎉

**Comment by SaraVieira** [comment]
DEPLOYED 🚀

**Comment by kutyel** [comment]
Nice! Although with your branch strategy I think github didn’t recognize any commit as mine and I don’t appear as a contributor to the project 😭🤣😅

**Pull request merged**